tests/test_synchronize.py

```diff
@@ -21,11 +21,16 @@ def test_semlock_failure():
     from loky.backend.synchronize import SemLock, sem_unlink
 
     name = "loky-test-semlock"
-    sl = SemLock(0, 1, 1, name=name)
-
-    with pytest.raises(FileExistsError):
-        SemLock(0, 1, 1, name=name)
-    sem_unlink(sl.name)
+    try:
```

Comment:
Maybe we could also add the following so that we nake sure it is cleaned up before hand?
```suggestion
    try:
        sem_unlink(name)
    except FileNotFoundError:
        pass
    try:
```